node/test/dialog.spec.ts

```diff
@@ -150,6 +194,8 @@ describe('dialog', async () => {
             return [200, { data: { lesson: navyIntegrityLesson } }];
           } else if ((reqBody.query as string).includes('q2')) {
             return [200, { data: { lesson: currentFlowLesson } }];
+          } else if ((reqBody.query as string).includes('q3')) {
```

Comment:
this hack for mapping the query from gql to Lesson data object has outlived its welcome.

We don't need a super elegant fix right now, but we should make it less egregious.

I recommend the following solution:

```typescript
const lessonById: Record<string, Lesson> = {
  q1: {...},
  q2: {...},
  q3: {...},
}

function findLessonForGqlQuery(query: string) : Lesson {
  lessonId = (query.includes("q1"))
    ? "q1"
    : (query.includes("q2")
    ? "q2"
    : (query.includes("q3")
    ? "q3"
    : "ok fix this properly with a regex already";
  return lessonById[lessonId]
}
```